user_management_schema.sql

```diff
@@ -0,0 +1,36 @@
+CREATE TABLE `sessions` (
+    `session_id` varchar(128) NOT NULL,
+    `expires` int(11) unsigned NOT NULL,
+    `data` text,
+    PRIMARY KEY (`session_id`)
+  ) DEFAULT CHARSET=utf8;
+  
+  CREATE TABLE IF NOT EXISTS `users` (
+      `id` int(5) NOT NULL AUTO_INCREMENT,
+      `email_id` varchar(225) NOT NULL,
+      `password` varchar(33) NOT NULL,
+      `is_active` BOOLEAN DEFAULT 1,
+      `is_admin` BOOLEAN DEFAULT 0,
+      `bucket_id` int(11) DEFAULT 0 NOT NULL,
+      PRIMARY KEY (`id`)
+    ) ENGINE=InnoDB  DEFAULT CHARSET=latin1 AUTO_INCREMENT=4;
+  
+  CREATE TABLE `buckets` (
+      `bucket_id` int(11) NOT NULL,
+      `value` text NOT NULL,
+      PRIMARY KEY (`bucket_id`)
+    ) DEFAULT CHARSET=utf8;
+  
+  CREATE TABLE IF NOT EXISTS `records` (
```

Comment:
kindly use `IF NOT EXISTS` with all of them, for consistency.

Reply:
done.